agent/exploits/cve_2024_6387.py

```diff
@@ -0,0 +1,100 @@
+"""Agent Asteroid implementation for CVE-2024-6387."""
+
+from threading import Lock
```

Comment:
```suggestion
import threading
```

https://google.github.io/styleguide/pyguide.html#22-imports